src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -61,17 +71,33 @@ export const TravelDetailsMapScreenComponent = ({
       );
       if (vehicle && vehicle !== followVehicleMapPoint) {
         setFollowVehicleMapPoint(vehicle.location);
```

Comment:
Shouldn't `setFollowVehicleMapPoint` in a `useEffect` which has `followVehicleMapPoint` in its dependency list.